### PR TITLE
MNT split and add exercise in cross-valdiation

### PR DESCRIPTION
**Pull request by glemaitre**

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
Should this stay as a Draft PR, or is it sufficiently advanced that we merge it, even if it is imperfect?

**Comment by GaelVaroquaux** [comment]
Given that we are currently working at a rapid pace, I am in general in favor of early merging when possible.

File: python_scripts/cross_validation_grouping.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# %% [markdown]
```
Comment (GaelVaroquaux) [inline python_scripts/cross_validation_grouping.py:1]: Should we have a title? It seems so to me.

File: python_scripts/cross_validation_ex_05.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+
```
Comment (GaelVaroquaux) [inline python_scripts/cross_validation_ex_05.py:1]: I think that we need a bit of text in the beginning of the file to describe what this is about

Comment (glemaitre) [reply python_scripts/cross_validation_ex_05.py:1]: Yes I am working on the example right now. more to come ;)

**Pull request merged**